samples_pipelines/azureml-designer-demo/README.md

```diff
@@ -0,0 +1,28 @@
+# azureml-designer-demo
+
+#### Overview
+In this repository, we customize some modules used on the Azure Machine Learning designer to reproduce the model of [fastText](https://arxiv.org/pdf/1607.01759.pdf). 
+If you have access to [Visual Studio Codespaces](https://visualstudio.microsoft.com/services/visual-studio-codespaces/), then you do not need to install the development tools and configure the environment. Because, they will be prepared automatically. Refer to ``` .devcontainer/how-to-use-Dockerfile.md ``` for more details about how to build your own docker image use on Codespaces.
+
+#### Module Description
+1. ```split_data_txt``` process the input data for train, evaluation, and score
+
+2. ```split_data_txt_for_parallel``` process the input data for train, evaluation, and score. This module organizes dataset according to demand of the parallel module
```

Comment:
split_data_txt_for_parallel no need to have the 'for', like fasttext_score_parallel